terraform/modules/rds/variables.tf

```diff
@@ -30,7 +36,8 @@ variable "enable_primary_deletion_protection" {
 
 variable "enable_replica_deletion_protection" {
   type        = bool
-  description = "Whether the replica database should have deletion protection enabled"
+  description = "Whether the replica database (if create_replica is true) should have deletion protection enabled"
+  default     = true

```

Comment:
What is the benefit of setting the default here?

Reply:
My reasoning here was it meant not needing to specify a value when there is no replica (which is a weird thing to have to do) - annoyingly you can't get interesting conditions that reference other variables, but true seems like a reasonable default here if you do have a replica and don't specify.

Reply:
Yup, that makes sense